### ISSUE TITLE
Automatic task sorting

### ISSUE THREAD

**Issue by Froloket64** [SleepySwords/do_todo#43]
So, I think it would be pretty nice to have a feature to automatically sort tasks as they are added by their priority.
However, I'm not sure how we should implement this. Should we replace the existing <kbd>s</kbd> and <kbd>S</kbd> key toggles for one-off sorting to toggle automatic sorting, or we should add this as a separate keybind?

Additional question: do you think lowest-to-highest priority sorting is even useful? I mean, it's easier to implement, but is it really needed. If not, we could replace, say, <kbd>S</kbd> to be _automatic_ sort, making <kbd>s</kbd> _one-off_ highest-to-lowest sort?

**Comment by SleepySwords** [comment]
I personally don't have a use for lowest-to-highest, so it should be fine to replace it, assuming that you also don't use it. I think that <kbd>S</kbd> for automatic sort and <kbd>s</kbd> for one-off sort would be nice. It might be confusing if it were accidentally set, but I think that should mostly be fine.

**Comment by Froloket64** [comment]
Yeah, I don't use low-to-high either, so I think replacing <kbd>S</kbd> with automatic is the way to go.

> It might be confusing if it were accidentally set, but I think that should mostly be fine.

I mean, can it really do any harm? Probably not :) But of course it should be off by default.
